queries/index.ts

```diff
@@ -15,6 +15,8 @@ export enum Views {
   VotesDiffsWithAverage = "votes_diff_with_average",
   Submissions = "submissions_view",
   VoteBreakdownBySong = "vote_breakdown_by_song",
+  PublicSignups = "public_signups",
+  PublicSubmissions = "public_submissions",
```

Comment:
So you added the PublicSignUps, and PublicSubmissions to this Enum for use in the [username] route. Why do you have to use an enum here?

Reply:
The idea is that I don't want to select strings, I want to select enum values that represent strings. Reason being, it's way more typesafe. 

I could also pull this off by properly configuring my Supabase instance, but...